src/project/view/exportdialogmodel.cpp

```diff
@@ -234,7 +243,11 @@ void ExportDialogModel::setExportType(const ExportType& type)
     }
 
     m_selectedExportType = type;
-    emit selectedExportTypeChanged(type.toMap());
+    m_exportPath = "";
+
+    QMetaObject::invokeMethod(qApp, [this, type]() {
+        emit selectedExportTypeChanged(type.toMap());
+    }, Qt::QueuedConnection);
```

Comment:
I wonder if it's still necessary to use `QMetaObject` etc here, now that https://github.com/musescore/MuseScore/pull/16837 has been merged